src/components/Icon/Icon.svelte

```diff
@@ -11,6 +12,10 @@
     case 'eye':
       icon = eye
       break
+
+    case 'star':
```

Comment:
À la longue, créer un dictionnaire serais mieux si il y a beaucoup d'Icon, mais pour l'instant le switch est plus adapté :+1:
J'y aurais pas pensé au switch !

Reply:
You're totally right 👍 on next icon addition we will change it to an object 👍

Reply:
I suppose we can wait for more than the next addition, I think the threshold to be 4 or 5 :wink: